styles.css

```diff
@@ -88,84 +98,191 @@ h3 {
     width: 60px;
     height: 60px;
     bottom: 80px;
-    right: 0;
+    border-radius: 5px;
+    right: 18px;
     z-index: 10;
 }
 
+.side-tab img {
+    position: absolute;
+    left: 7px;
+    top: 6px;
+}
+
+.side-tab:hover {
+    background-color: #ABB2B9;
+}
+
 /*=====================SECTION - ABOUT=======================*/
 
 section {
     min-height: 390px;
+    background-color: #EAECEE;
     position: relative;
 }
-.cloud-video {
-    position: absolute;
-    left: 0;
-    top: 0;
-    width: 100vw;
-    background-size: cover;
-    z-index: -1;
-}
 
 section div {
     padding: 70px 0 0;
     font-size: 1.5rem;
+    line-height: 150%;
     overflow: auto;
+    margin: 0 auto;
+    width: 90%;
+}
+
+.icons {
+    margin: 0 auto;
+    padding: 20px;
+}
+
+.icons div {
+    display: inline-block;
+    width: 19%;
+    height: 100%;
+}
+
+section img {
+    height: 100px;
+    width: 100px;
+    margin: 0 auto;
 }
 
 /*=====================MAIN - PORTFOLIO========================*/
 
 main {
+    width: 100%;
     overflow: auto;
-  padding-top: 60px;
 }
 
-main div {
-    width: 1600px;
+.portfolio-box {
+    width: 1700px;
     overflow: auto;
     margin: 0 auto;
-    padding: 30px;
+    margin-top: 80px;
 }
 
 article {
-    height: 500px;
-    width: 450px;
+    height: 400px;
+    width: 33%;
     float: left;
-    margin: 10px;
+    margin-bottom: 65px;
 }
 
-article div {
-    width: 410px;
+.img-holder {
+    width: 390px;
     height: 260px;
     margin: 0 auto;
```

Comment:
could you combine margin:auto for several selectors in one?

Reply:
maybe, but i've tried to keep the css in separate sections for easy navigation so i might leave them as they are.